povertymapping/nightlights.py

```diff
@@ -357,9 +358,12 @@ def generate_clipped_raster(
         process_suffix=process_suffix,
         vcmcfg=vcmcfg
     )
-    viirs_zipped_filename = urlclean(viirs_url)
+    parsed_url = urlparse(viirs_url)
+    viirs_zipped_filename = Path(os.path.basename(parsed_url.path)).name
```

Comment:
Clarifying if I got it right. The cause of the bug was that the `viirs_zipped_filename` was incorrectly inferred from `viirs_url`?

Just curious because it seems to work with no problem on my laptop. What edge case causes this to happen?